package.json

```diff
@@ -39,7 +40,7 @@
     "walk-sync": "^0.3.1"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "6.* || 8.* || 10.* || >= 12.*"
```

Comment:
This is a breaking change, we should introduce this as another PR.

Reply:
Looks like you amended the commit to remove this change. I updated the commit message to match.